libr/bin/format/elf/elf.c

```diff
@@ -1459,7 +1442,9 @@ static size_t get_size_rel_mode(Elf_(Xword) rel_mode) {
 
 static size_t get_num_relocs_dynamic_plt(ELFOBJ *bin) {
 	if (bin->dyn_info.dt_pltrelsz) {
-		return bin->dyn_info.dt_pltrelsz / get_size_rel_mode (bin->dyn_info.dt_pltrel);
+		const size_t size = bin->dyn_info.dt_pltrelsz;
```

Comment:
I think this should be `ut64`, otherwise on 32bit machine it's going to fail.

Reply:
this is still not addressed.

Reply:
pltrelsz is a small number, could be 8, 32 or 2. making it 64bit wide wont fix anything

Reply:
I don't see where is the check that pltrelsz should be 8, 2 or 32, since pltrelsz comes directly from the binary. Also, in glibc_elf.h you see:
```
typedef uint64_t Elf32_Xword;
typedef uint64_t Elf64_Xword;
```
So I don't see why this should be cast to size_t for no reason.